src/test/relation_engine_client/test_integration.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+import os
+from uuid import uuid4
+
+from src.relation_engine_client import REClient
+from src.relation_engine_client.exceptions import RERequestError, RENotFound
+
+_API_URL = os.environ.get('RE_API_URL', 'http://localhost:5000')
+_VERT_COLL = 'test_vertex'
+_EDGE_COLL = 'test_edge'
+_TOK_ADMIN = 'admin_token'
```

Comment:
These are coming from the auth mock? Should document if so, going to be hard to understand for the next guy otherwise